app/views/materials/compare.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="jumbotron">
+  <form>
+    <% @materials.each_with_index do |material, index| %>
+    <% value = params["material_#{index}_uuid"] %>
+      <div class="row">
+        <div class="col mb-4">
+          <%= label_tag "material_#{index+1}_uuid" %>
+          <%= text_field nil, "material_#{index}_uuid", value: value, class: "form-control", placeholder: "UUID" %>
```

Comment:
If you give this the same name as the other inputs then I think rails will create an array for you. For example if the name is changed to `material_ids[]`, `params[:material_ids]` will be an array.

Reply:
Hopefully this will simplify a few things.

Reply:
I wanted to use arrays for the params, but I read somewhere that arrays could not be used with GET requests, but re-reading that same post that was not the case. I'll switch to that then.